typescript/sdk/src/core/message.ts

```diff
@@ -366,11 +371,4 @@ export class HyperlaneMessage {
   get transactionHash(): string {
     return this.dispatch.event.transactionHash;
   }
-
-  /**
-   * The messageId committed to the tree in the Outbox contract.
-   */
-  get id(): string {
```

Comment:
ditto, *could* compute this from the dispatch event rather than finding the other log and passing it in the constructor